Sources/Core/Countries.swift

```diff
@@ -0,0 +1,105 @@
+//
+//  Countries.swift
+//  Country Picker
+//
+//  Created by Uwe Tilemann on 07.03.24.
+//
+
+import Foundation
+
+public struct Country {
+    public static var `default`: [Country] = loadJSON("Countries")
+
+    public let code: String
+    public let label: String
+    public let callingCode: UInt
+    public let states: [State]?
+
+    public struct State {
+        public let code: String
+        public let label: String
+    }
+    public init(code: String, label: String, callingCode: UInt, states: [State]? = nil) {
+        self.code = code
+        self.label = label
+        self.callingCode = callingCode
+        self.states = states
+    }
+    public var flagSymbol: String? {
+        code.flagSymbol
+    }
+}
+
+extension Country.State: Decodable {
+    private enum CodingKeys: String, CodingKey {
+        case code
+        case label
+    }
+    public init(from decoder: Decoder) throws {
+        let container = try decoder.container(keyedBy: CodingKeys.self)
+        self.code = try container.decode(String.self, forKey: .code)
+        self.label = try container.decode(String.self, forKey: .label)
+   }
+}
```

Comment:
Das sollte nicht notwendig sein. Ein `Decodable` in Zeile 18 sollte ausreichen. Hattest du Probleme beim Parsing?

Reply:
https://github.com/snabble/snabble-phoneauth-ios-sdk/pull/12/commits/f9ea062f4f0c9549cfee688cbb0def000d263f7f